packages/react-storage/src/components/StorageBrowser/types.ts

```diff
@@ -44,6 +44,7 @@ export interface CreateStorageBrowserInput {
   actions?: StorageBrowserActions;
   config: Config;
   components?: Components;
+  ErrorBoundary?: React.ComponentClass;
```

Comment:
Can we create and export a `ErrorBoundaryType` interface from using the types in the [default `ErrorBoundary` file](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-storage/src/components/StorageBrowser/ErrorBoundary/ErrorBoundary.tsx)?